### PR TITLE
Dustin simplify commands

### PR DESCRIPTION
**Pull request by wagnerd97**
Sorry this branch contains a few things beyond the branch name:
first it simplifies the yarn installation and adds a yarn run example line.
it also cleans up the printing of data to be more legible. particularly for large datasets. 
it increase max csp packet size to accommodate housekeeping packet size.
lastly it adds the dictionaries needed to send and receive housekeeping requests.

### CODE REVIEW

**Review by ArashYzd — APPROVED** [review]
Looks great! While you're at it, Could you also add `--enable-crc32` to the configuration in the build? Thanks.

**Comment by wagnerd97** [comment]
@ArashYzd Where did you want that added, sorry?

File: src/cli.py [hunk 1]
```diff
@@ -34,7 +34,9 @@ def cli():
         try:
             server, port, toSend = gs.getInput(prompt='to send: ')
             resp = gs.transaction(server, port, toSend)
-            print(resp)
+
+            #print(resp)
```
Comment (arrooney) [inline src/cli.py:38]: delet this

File: src/groundStation/groundStation.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def transaction(self, server, port, buf):
             libcsp.conn_sport(conn),
             data,
             length)
-
+
```
Comment (arrooney) [inline src/groundStation/groundStation.py:167]: delet this

File: refresh.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (arrooney) [inline refresh.sh:1]: perhaps there's a better name for this file. Maybe `install.sh` or `setup_env.sh` or something

**Review by arrooney — APPROVED** [review]

**Pull request merged**